image/bmp.ksy

```diff
@@ -16,7 +16,7 @@ meta:
     wikidata: Q192869
   endian: le
   license: CC0-1.0
-  ks-version: 0.8
+  ks-version: 0.9
```

Comment:
Ok, this change makes it slightly more complicated: if we'll merge it as is, it will fail compilation on https://formats.kaitai.io/ at all, as it currently only fetches "latest stable" (v0.8) to use.

Ideal solution would be (finally) releasing 0.9 to make it "officially stable", and continue using that — otherwise, we'll need to have something clever like bringing multiple ksc versions into build...